src/OrchardCore/OrchardCore.Email.Abstractions/MailMessageBody.cs

```diff
@@ -0,0 +1,17 @@
+namespace OrchardCore.Email;
+
+/// <summary>
+/// Represents a body for <see cref="MailMessage"/>.
+/// </summary>
+public class MailMessageBody
+{
+    /// <summary>
+    /// Gets or sets the body in plain text format.
+    /// </summary>
+    public string Text { get; set; }
+
+    /// <summary>
+    /// Gets or sets the body in HTML format.
+    /// </summary>
+    public string Html { get; set; }
```

Comment:
Maybe `IHtmlContent` instead to be better typed? With a convenience conversion method to/from string.

Reply:
For simplicity, I prefer `string` instead, like other email messaging APIs

Reply:
Yeah but `string` is really overused. `IHtmlContent` is for when you want to store a string that's actually HTML.